config/packages/cache.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services
+        https://symfony.com/schema/dic/services/services-1.0.xsd
+        http://symfony.com/schema/dic/symfony
+        https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config>
+        <framework:cache app="cache.adapter.filesystem"
```

Comment:
that's an invalid config. It should look like that:

```xml
<framework:cache name="app" adapter="cache.adapter.filesystem" />
<framework:cache name="system" adapter="cache.adapter.system" />
```

Prototyped array nodes work in XML by repeating the tag, and the key is hold by an attribute (as XML does not have any way to represent a map, as attribute names and tag names cannot be dynamic in XSD). Btw, that's the reason why the method of the `Config\Definition` system to mark a prototyped array node as a map is named `useAttributeAsKey` and accept an argument: it relates to the fact that XML needs an attribute to store the key.

Reply:
Note that I haven't checked the actual config to be sure that the attribute holding the key is `name` in that case. So my config might be wrong. But `name` is the most used in the Symfony configurations so that was an easy guess.

Reply:
Hm.. I think you are considering the `pools` key. 

Here is some Yaml that I would like to convert to XML. 

```yaml
framework:
    cache:
        app: cache.adapter.filesystem
        system: cache.adapter.system

        pools:
            acme:
                adapter: cache.app
            redis:
                adapter: cache.adapter.redis
```

Reply:
Also note, what I currently have written in this PR is copied from the docs: https://symfony.com/doc/current/cache.html#configuring-cache-with-frameworkbundle

Reply:
hmm, looks like the XSD has been created putting those a child elements rather than attributes (which is not consistent with other parts of Symfony): https://github.com/symfony/symfony/blob/8361713367e2735fd9b63cd2b6661411a0339811/src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd#L275-L286